### PR TITLE
[NCL-8506] Provide Build Log link on Status Icon

### PR DESCRIPTION
**Pull request by patrikk0123**

### CODE REVIEW

File: src/components/BuildStatusIcon/BuildStatusIcon.tsx [hunk 1]
```diff
@@ -51,16 +52,20 @@ export const BuildStatusIcon = ({ build, long }: IBuildStatusIcon) => {
     ((build as Build).attributes?.POST_BUILD_REPO_VALIDATION === 'REPO_SYSTEM_ERROR' ||
       (build as Build).attributes?.PNC_SYSTEM_ERROR === 'DISABLED_FIREWALL');
 
+  const buildLogLink = selectedIconData.failed ? `/builds/${build.id}/alignment-log` : `/builds/${build.id}/build-log`;
```
Comment (matedo1) [inline src/components/BuildStatusIcon/BuildStatusIcon.tsx:55]: @jomrazek is this condition valid?

File: src/components/BuildStatusIcon/BuildStatusIcon.module.css [hunk 2]
```diff
@@ -64,6 +64,10 @@
   vertical-align: middle;
 }
 
+.build-log-link img {
+  display: block;
```
Comment (matedo1) [inline src/components/BuildStatusIcon/BuildStatusIcon.module.css:68]: `display: inline-block`

**Review by matedo1 — CHANGES_REQUESTED** [review]

File: src/components/BuildStatusIcon/BuildStatusIcon.module.css (hunk 2 above)
Comment (patrikk0123) [reply src/components/BuildStatusIcon/BuildStatusIcon.module.css:68]: no, it has  to be block

File: src/components/BuildStatusIcon/BuildStatusIcon.tsx (hunk 1 above)
Comment (patrikk0123) [reply src/components/BuildStatusIcon/BuildStatusIcon.tsx:55]: probably some other condition would suit better, I put there some just as proof of concept

Comment (matedo1) [reply src/components/BuildStatusIcon/BuildStatusIcon.tsx:55]: We discussed another approach on PM.

File: src/components/BuildStatusIcon/BuildStatusIcon.tsx [hunk 3]
```diff
@@ -88,3 +92,22 @@ export const BuildStatusIcon = ({ build, long }: IBuildStatusIcon) => {
     </span>
   );
 };
+
+interface IBuildLogLink {
```
Comment (matedo1) [inline src/components/BuildStatusIcon/BuildStatusIcon.tsx:96]: `IBuildLogLinkProps`

File: src/components/BuildStatusIcon/BuildStatusIcon.tsx [hunk 4]
```diff
@@ -88,3 +92,22 @@ export const BuildStatusIcon = ({ build, long }: IBuildStatusIcon) => {
...
+  build: Build;
+}
+
+const BuildLogLink = ({ build, children }: PropsWithChildren<IBuildLogLink>) => {
+  const buildLogLink = useMemo(() => getAdequateBuildLogLink(build), [build]);
+
+  return (
+    <TooltipWrapper tooltip={<span>{buildStatusData[build.status!].tooltip}. Click to open the log</span>}>
```
Comment (matedo1) [inline src/components/BuildStatusIcon/BuildStatusIcon.tsx:104]: `<span>` should not be needed

**Review by matedo1 — APPROVED** [review]
Approved, just minor comments

**Pull request merged**